tests/org.eclipse.jface.tests/src/org/eclipse/jface/tests/viewers/AbstractTreeViewerTest.java

```diff
@@ -151,6 +151,54 @@ public void testExpandToLevel() {
 		assertNotNull("first3 is visible", fViewer.testFindItem(first3));
 	}
 
+	public void testAutoExpandTrivialPath() {
+		TestElement modelRoot = TestElement.createModel(5, 1);
+		TestElement trivialPathRoot = modelRoot.getFirstChild();
+		fViewer.setInput(modelRoot);
+
+		fTreeViewer.setAutoExpandTrivialPathsLevel(1);
+		fTreeViewer.setExpandedState(trivialPathRoot, true);
+
+		assertTrue("The expanded widget child is expanded", fTreeViewer.getExpandedState(trivialPathRoot));
```

Comment:
Applies to all assertion messages: the message describe the failure state, not the success state, i.e., the messages have to be "inverted".